lib/gulpDepcheck.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const _ = require('lodash'),
+      dc = require('depcheck'),
```

Comment:
There will be a variable naming conflict between the global `depcheck` and the `const depcheck` within the `gulpDepcheck` scope. Do you have a different preferred name, or another workaround?